test/Crowdsale.js

```diff
@@ -1,5 +1,8 @@
+import moment from 'moment'
```

Comment:
I've removed moment.js dependencies from the crowdsale code and intoduced a small duration helper.